change/@ni-nimble-blazor-0f03cea0-79ac-44ba-983f-26623b2691f9.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We should add some CONTRIBUTING docs that describe this new infrastructure and state our policies about when Playwright tests are needed / desirable. (You can probably lift a lot of it from your PR description)

Reply:
Update the Nimble Blazor CONTRIBUTING.md doc with info about these acceptance tests.

Reply:
The docs look great!